html-forms/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <div>
+      <form method="get">
+        <label for="email">Email</label>
+        <div><input type="text" id="email" name="email"></div>
+      </form>
+    </div>
+    <div>
+      <label for="selection">Why is coding cool?</label>
+    </div>
+    <div>
+      <select id="selection" name="selection">
+        <option value="select an option">Select an option</option>
+        <option value="because it's interesting">Because it's interesting</option>
+        <option value="because it's fun">Because it's fun</option>
+        <option value="because it's powerful">Because it's powerful</option>
+      </select>
+    </div>
+    <div>
+      <p>
+        <label for="radio"> Favorite Language (choose one)</label>
+      </p>
+    </div>
+    <div>
+      <label id="radio">
+        <input type="radio" name="language" value="html"> HTML
+        <input type="radio" name="language" value="css"> CSS
+        <input type="radio" name="language" value="javascript"> Javascript
+      </label>
+    </div>
+    <p>
+      <label for="checkbox">Would you like to sign up for more information?</label>
+    </p>
+    <div>
+      <input type="checkbox" name="y/n" value="sign me up!"> Sign me up!
+    </div>
+    <label for="Comments">Comments</label>
```

Comment:
make sure that the `for` attribute value matches, you don't want to use a capital `C` here.